exercises/go-1-a/.meta/example_range_array.go

```diff
@@ -0,0 +1,31 @@
+package go1a
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+var cutoffs = []struct {
```

Comment:
Or using maps? [int]string

Reply:
In a map the iteration order is pseudo-random. I think that would randomly rail the tests.

Reply:
I would consider that a bug in the testing, unless the instructions specifically state a specific order

Reply:
@ErikSchierboom: To be clarified

Reply:
> I would consider that a bug in the testing, unless the instructions specifically state a specific order

I indeed didn't use a map for the iteration order issue. I've updated the description to make it clear that one should use descending unit amounts.